src/backend/joanie/core/models/products.py

```diff
@@ -397,6 +420,28 @@ def can_edit(self):
         """Return True if the order group can be edited."""
         return not self.orders.exists()
 
+    @property
+    def available_seats(self) -> int | None:
+        """Return the number of available seats on the order group."""
+        if self.nb_seats is None:
+            return None
+        return self.nb_seats - self.get_nb_binding_orders()
+
+    @property
+    def is_enabled(self):
+        """
+        Returns boolean whether the order group is enabled based on its activation status
+        and time constraints.
+        """
+        if not self.is_active:
+            return False
+
+        now = timezone.now()
+        start = self.start or datetime.min.replace(tzinfo=timezone.utc)
+        end = self.end or datetime.max.replace(tzinfo=timezone.utc)
```

Comment:
Maybe simpler with this ?
```suggestion
        now = timezone.now()
        start = self.start or now
        end = self.end or now
```